src/main/java/com/autotune/database/table/KruizeRecommendationEntry.java

```diff
@@ -27,6 +27,7 @@ public class KruizeRecommendationEntry {
     private String cluster_name;
     @JdbcTypeCode(SqlTypes.JSON)
     private JsonNode extended_data;
+    private String experiment_type;
```

Comment:
please see if it can be camelCase

Reply:
Using camelCase with experiment entries can cause readability issues, as SQL is case-insensitive, and it would become `experimenttype` in queries, which isn't very readable. I think that's why snake_case is being used here. I haven't updated it to camelCase yet. What do you think—should I go ahead and make the change?

Reply:
ok seems fine